### PR TITLE
chore(libs): Update all the npm packages

### PR DESCRIPTION
**Pull request by vikalpj**

### CODE REVIEW

File: cookiecutter.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    , "github_username": "Fueled"
-    , "github_reponame": ""
-    , "project_description": "add a short project description here"
-    , "timezone": "UTC"
-    , "version": "0.1.0"
-    , "api_base_url": "http://localhost:8000/api/v1/"
-    , "api_auth_url": "auth"
+    "base_app_name": "Hello World",
+    "repo_name": "Hello World",
+    "project_name": "Hello World",
```
Comment (theskumar) [inline cookiecutter.json:2]: We can use something like this https://github.com/Fueled/django-init/blob/master/cookiecutter.json#L4 here.

**Review by theskumar — CHANGES_REQUESTED** [review]

**Comment by theskumar** [comment]
Looks good otherwise for merge! 👍

**Comment by theskumar** [comment]
We should also update Readme about updated angular and can add `CHANGELOG.md` file.

File: cookiecutter.json (hunk 1 above)
Comment (vikalpj) [reply cookiecutter.json:2]: done :)

File: {{cookiecutter.repo_name}}/app/scripts/router.js [hunk 2]
```diff
@@ -53,14 +53,14 @@ app.run(['$rootScope', '$state', 'CurrentUser', function($rootScope, $state,  Cu
...
+  $rootScope.$on('$stateChangeSuccess', function(event, toState){
+    console.log('[ AUTH ] : State change success.... to ' + toState.name);
   });
 
-  $rootScope.$on('$stateChangeStart', function(event, toState, toParams, fromState, fromParams){
-    console.log('[ AUTH ] : State change initiated.... to '+toState.name+' from '+fromState.name );
-    if(toState.name == 'login'){
-      return
+  $rootScope.$on('$stateChangeStart', function(event, toState, toParams, fromState){
+    console.log('[ AUTH ] : State change initiated.... to ' + toState.name + ' from ' + fromState.name );
```
Comment (glemmaPaul) [inline {{cookiecutter.repo_name}}/app/scripts/router.js:61]: Might want to change this to $logger?

**Comment by glemmaPaul** [comment]
Looks good to me! Don't mind the netlify builds, was trying it out

File: {{cookiecutter.repo_name}}/app/scripts/router.js (hunk 2 above)
Comment (theskumar) [reply {{cookiecutter.repo_name}}/app/scripts/router.js:61]: I think we should do that in another PR :) I like the suggestion.

**Pull request merged**